lang/en.json

```diff
@@ -399,26 +399,28 @@
             "no_items": "The items you've recently looked at on our site will appear below."
         },
         "wishlists": {
-            "heading": "Wish lists",
-            "tag_line": "Fill in the form below to create a wish list. Click the \"Save\" button when you're done.",
-            "new": "New wish List",
-            "you_have_none": "You have no wish lists, add one now.",
-            "empty_wishlist": "Your wish list is empty. When you add items to your wish list they will appear here.",
+            "heading": "Wish Lists",
+            "tag_line": "Fill in the form below to create a Wish List. Click the \"Save\" button when you're done.",
```

Comment:
Is "Click the "Save" button when you're done" still accurate? Specifically the "Save." It looks like the button will get "Create New Wish List."

Reply:
good catch, the button in this case is actually "Create Wish List" without the "New".  i've updated the copy, replacing "Save" with "Create Wish List".